src/mscorlib/src/System/Threading/Timer.cs

```diff
@@ -78,6 +78,7 @@ private static int TickCount
             [SecuritySafeCritical]
             get
             {
+#if !FEATURE_PAL
                 if (Environment.IsWindows8OrAbove)
```

Comment:
For CoreCLR, we always assume Win8 or above. My reason for ifdefing this out was that the BCL thinner will tree-shake QueryUnbiasedInterruptTime because of it will be statically unreachable. It will make it easier to audit that all mscorlib.dll P/Invokes are implemented.

The remaining call sites are in globalization.